### PR TITLE
[refactor](dialect) make http sql converter plugin and audit loader as builtin plugin

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

Followup #28890

Make HttpSqlConverterPlugin and AuditLoader as Doris' builtin plugin.
To make it simple for user to support sql dialect and using audit loader.

- HttpSqlConverterPlugin

	1. By default, there is nothing changed.
	2. There is a new global variable `sql_converter_service`, default is empty, if set, the HttpSqlConverterPlugin will be enabled

		`set global sql_converter_service = "http://127.0.0.1:5001/api/v1/convert"`

- AuditLoader

	1. By default, there is nothing changed.
	2. There is a new global variable `enable_audit_plugin`, default is false, if set to true, the audit loader plugin will be enable.
	3. Doris will create `audit_log` in `__internal_schema` when startup
	4. If `enable_audit_plugin` is true, the audit load will be inserted into `audit_log` table.
	5. 3 other global variables related to this plugin:

		- `audit_plugin_max_batch_interval_sec`: The max interval for audit loader to insert a batch of audit log.
		- `audit_plugin_max_batch_bytes`: The max batch size for audit loader to insert a batch of audit log.
		- `audit_plugin_max_sql_length`: The max length of statement in audit log

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
run buidall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java [hunk 1]
```diff
@@ -83,6 +113,7 @@ public void run() {
         Database database = op.get();
         modifyTblReplicaCount(database, StatisticConstants.STATISTIC_TBL_NAME);
         modifyTblReplicaCount(database, StatisticConstants.HISTOGRAM_TBL_NAME);
+        modifyTblReplicaCount(database, AuditLoaderPlugin.AUDIT_LOG_TABLE);
```
Comment (dutyu) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java:116]: Should we move `StatisticConstants.STATISTIC_INTERNAL_TABLE_REPLICA_NUM` to this file and rename it to `INTERNAL_TABLE_REPLICA_NUM`?

File: fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java [hunk 2]
```diff
@@ -104,7 +110,22 @@ public final class GlobalVariable {
     @VariableMgr.VarAttr(name = SHOW_FULL_DBNAME_IN_INFO_SCHEMA_DB, flag = VariableMgr.GLOBAL)
     public static boolean showFullDbNameInInfoSchemaDb = false;
 
-    // Don't allow to create instance.
+    @VariableMgr.VarAttr(name = SQL_CONVERTER_SERVICE, flag = VariableMgr.GLOBAL)
+    public static String sqlConverterService = "";
```
Comment (dutyu) [inline fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java:114]: Maybe `sqlConverterServiceUrl` fits the meaning better? just my personal preference

**Review by dutyu — CHANGES_REQUESTED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java [hunk 3]
```diff
@@ -439,8 +433,10 @@ private Object executeWithClusterToken(HttpServletRequest request, String db,
 
             return redirectView;
         } catch (Exception e) {
-            LOG.warn("Failed to execute stream load with cluster token, {}", e);
+            LOG.warn("Failed to execute stream load with cluster token, {}", e.getMessage(), e);
```
[the remark this answers is not in the document]
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/LoadAction.java:436]: my way is more friendly for `grep`

File: fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java (hunk 1 above)
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java:116]: I will try to refactor this later, there are some other things need to be done

File: fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java (hunk 2 above)
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/qe/GlobalVariable.java]: done

**Comment by morningman** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java (hunk 1 above)
Comment (dutyu) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/InternalSchemaInitializer.java:116]: got it

**Review by dutyu — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]

**Pull request merged**